src/content/LandingPage/LandingPage.js

```diff
@@ -1,7 +1,102 @@
 import React from 'react';
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  Button,
+  Tabs,
+  Tab,
+} from 'carbon-components-react';
+
+const props = {
+  tabs: {
+    selected: 0,
+    triggerHref: '#',
+    role: 'navigation',
+  },
+  tab: {
+    href: '#',
+    role: 'presentation',
+    tabIndex: 0,
+  },
+};
 
 const LandingPage = () => {
-  return <div>LANDING PAGE</div>;
+  return (
+    <div classname="bx--grid bx--grid--full-width landing-page">
```

Comment:
This should be `<div className="bx--grid bx--grid--full-width landing-page">`

Reply:
That explains why it didn't look quite right. Thank you!